.circleci/base-install.sh

```diff
@@ -5,12 +5,14 @@ DIR=$(dirname "$0")
 cd "$DIR/.."
 
 # npm install just enough to run these scripts
-npm ci --ignore-scripts --no-optional --only=prod
+npm i --ignore-scripts --no-optional --only=prod --no-package-lock
```

Comment:
Do we care that this doesn't first remove node_modules like `ci` does?

Reply:
this runs in circleci so there's no existing node_modules